app/controllers/api/account.php

```diff
@@ -2,7 +2,7 @@
 
 use Ahc\Jwt\JWT;
 use Appwrite\Auth\Auth;
-use Appwrite\Auth\Validator\Password;
+use Appwrite\Auth\Validator\PasswordNew;
```

Comment:
Why the rename?

Reply:
The new class is just a draft, It will be called Password.php once we finalise the changes